src/main/java/com/lorepo/icplayer/client/module/text/WCAGUtils.java

```diff
@@ -98,8 +98,12 @@ private static String getElementTextElementContent (NavigationTextElement elemen
 			return "";
 		}
 		TextElementDisplay _element = (TextElementDisplay) element;
-		return _element != null ? _element.getWCAGTextValue() : "";
+		return _element != null ? decodeString(_element.getWCAGTextValue()) : "";
```

Comment:
Czy ta zmiana nie jest związana z innym ticketem ?

Reply:
Nie, jest to rozwiązanie do 2 punktu

Reply:
rozwiązanie punktu 2